### PR TITLE
Adding Scale to Universe

### PR DESCRIPTION
**Pull request by gisjedi**
Scale enables on-demand, near real-time, automated processing of large datasets (satellite, medical, audio, video, ...) using a dynamic bank of algorithms. Algorithm execution is seamlessly distributed across thousands of CPU cores. Docker provides algorithm containerization.

http://ngageoint.github.io/scale/

### CODE REVIEW

File: repo/packages/S/scale/0/resource.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (ryadav88) [inline repo/packages/S/scale/0/resource.json]: Please change the order. Images first then assets info

File: repo/packages/S/scale/0/resource.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "assets": {
+    "container": {
+      "docker": {
+         "scale": "docker.io/geoint/scale:4.0.0",
```
Comment (ryadav88) [inline repo/packages/S/scale/0/resource.json]: change to 'geoint/scale:4.0.0'. same for db and logstash

File: repo/packages/S/scale/0/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "packagingVersion": "3.0",
+  "minDcosReleaseVersion": "1.7",
+  "scm": "https://github.com/ngageoint/scale.git",
+  "postInstallNotes": "The Scale DCOS Service has been successfully installed!\n\n\tDocumentation: https://ngageoint.github.io/scale/\n\tIssues: https://github.com/ngageoint/scale/issues",
+  "maintainer": "http://www.nga.mil",
```
Comment (ryadav88) [inline repo/packages/S/scale/0/package.json]: change this to an email id where users can reach out for questions related to the package.

File: repo/packages/S/scale/0/package.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  "tags": [
+    "bigdata",
+    "batch",
+    "analytics",
+    "remotesensing",
+    "algorithms",
+    "geospatial"
+  ],
+  "framework": true,
+  "version": "4.0.0",
```
Comment (ryadav88) [inline repo/packages/S/scale/0/package.json]: version should be "scaleVersion-universePackageVersion". Ex: if Scale's version is 4.0.0 then version should be "4.0.0-0.0.1

File: repo/packages/S/scale/0/package.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  "postUninstallNotes": "The Scale Scheduler has been uninstalled and will no longer run. You may need to remove supporting services - scale-logstash, scale-webserver and scale-db.",
+  "preInstallNotes": "We recommend a minimum of three nodes with at least 4 CPU and 6GB of RAM available for the Scale services and running Scale jobs.\n\nElasticsearch is assumed to be running within your DCOS cluster. If you wish to use an externally hosted Elasticsearch cluster, specify one or more of the nodes in comma delimited format in the SCALE_ELASTICSEARCH_URLS variable. For quick-start purposes, Scale is bootstrapped with a Postgres database. This should NEVER be used for production purposes as it offers no underlying storage persistence. It can be replaced with an externally hosted Postgres by setting DB_HOST and associated settings appropriately.\n\nIf you are running DCOS 1.8 Enterprise Edition or higher, you will need to set the DCOS_OAUTH_TOKEN in the DCOS section of the Advanced Settings. This value can be found within the dcos.toml file under in the dcos_acs_token value on a system with an authenticated DCOS CLI.",
```
Comment (ryadav88) [inline repo/packages/S/scale/0/package.json]: Add this line first - "This DC/OS Service is currently EXPERIMENTAL. There may be bugs, incomplete features, incorrect documentation, or other discrepancies.". 
Refer: https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/O/openldap/3/package.json

Comment (ryadav88) [inline repo/packages/S/scale/0/package.json]: Highlight the fact that `Elasticsearch` is needed for this package. Maybe make it bold?

File: repo/packages/S/scale/0/package.json (hunk 3 above)
Comment (ryadav88) [inline repo/packages/S/scale/0/package.json]: add `website` field

File: repo/packages/S/scale/0/config.json [hunk 6]
```diff
@@ -0,0 +1,100 @@
+{
+  "properties": {
+    "scale": {
+      "properties": {
+        "framework-name": {
```
Comment (ryadav88) [inline repo/packages/S/scale/0/config.json]: change to `name`

File: repo/packages/S/scale/0/config.json [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+          "type": "string",
+          "description": "The name of the framework. MUST BE LOWERCASE!",
```
Comment (ryadav88) [inline repo/packages/S/scale/0/config.json]: change to `Name of this service instance.`

File: repo/packages/S/scale/0/config.json [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+          "default": "scale"
+        },
+        "vhost-name": {
```
Comment (ryadav88) [inline repo/packages/S/scale/0/config.json]: change to `virtual_host`

File: repo/packages/S/scale/0/config.json [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+          "type": "string",
+          "description": "Virtual Hostname for Marathon-LB webserver exposure outside cluster.",
+          "default": "scale.marathon.slave.mesos"
+        },
+        "docker-creds": {
```
Comment (ryadav88) [inline repo/packages/S/scale/0/config.json]: `docker-credentials`

File: repo/packages/S/scale/0/config.json [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+          "type": "string"
+        }
+      }
+    },
+    "webserver":{
+      "properties":{
+        "cpu": {
+          "description": "Allocation of CPU resources for Scale web server.",
+          "type": "integer",
+          "default": 1
```
Comment (ryadav88) [inline repo/packages/S/scale/0/config.json]: lets add `minimum` to both cpu and memory

**Comment by ryadav88** [comment]
@gisjedi here are the examples you can follow and create PR against this repo: https://github.com/dcos/examples

this example would finally land on the DC/OS documentation site here: https://dcos.io/docs/1.8/usage/tutorials/

**Comment by gisjedi** [comment]
Thanks for the links. I was in the process of finalized updates to the quick start hosted on GitHub Pages to target DCOS deployment.  I can take the same content and drop it into a pull request there as well. Hopefully, I'll have this wrapped up today.

**Comment by ryadav88** [comment]
great! thanks and for consistency, please follow the format from the other examples.

**Comment by gisjedi** [comment]
@ryadav88 I think everything is in order with the tutorial. Let me know if there is anything else you need from me to get this closed out. Thanks!

**Comment by ryadav88** [comment]
@gisjedi thanks for adding the example. This LGTM!

**Pull request merged**